subjects/printfirsthalf/README.md

```diff
@@ -0,0 +1,46 @@
+## printfirsthalf
+
+### Instructions
+
+Write a function called `PrintFirstHalf()` that takes a `string` as an argument and returns printed the first half of a `string`.
+
+- If the length of the `string` is odd round it down.
+- Prints the first half followed by a newline `\n`.
+- If the `string` is empty, return `Invalid Output`, followed by a newline `\n`.
+- If the `string` length is equal to one, return it, followed by a newline `\n`.
```

Comment:
Sounds like it asks to return the string length = 1, should ask "If the `string` length is equal to one, return the string (...)"

Reply:
It is fixed.